libs/akita/src/lib/entityStore.ts

```diff
@@ -58,6 +64,19 @@ export class EntityStore<S extends EntityState = any, EntityType = getEntityType
 
   constructor(initialState: Partial<S> = {}, protected options: Partial<StoreConfigOptions> = {}) {
     super({ ...getInitialEntitiesState(), ...initialState }, options);
+
+    this.ttlQueue.expired$.pipe(filter((ttl) => ttl.type === TTLType.Entity)).subscribe((ttl) => {
+      if (ttl.id && this.ids.includes(ttl.id)) {
+        this._setState((state) => {
+          const idsExpired = state.idsExpired || [];
+
+          return {
+            ...state,
+            idsExpired: idsExpired.includes(ttl.id) ? idsExpired : [...idsExpired, ttl.id],
```

Comment:
Do you think we need to add the property to the state? Another option is to add it to the instance.

Reply:
If it's part of the state it can be subscribed through `select` without introducing a new observable. In addition, expirations can be debugged and tracked through devtools. So I thought it would be fine to put it in the state itself.